src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div>
+      <div class="stars stars--0">
+        <div class="stars__star">1</div>
+        <div class="stars__star">2</div>
+        <div class="stars__star">3</div>
+        <div class="stars__star">4</div>
+        <div class="stars__star">5</div>
```

Comment:
no need to set numbers 
```suggestion
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
```